### ISSUE TITLE
Error Bad Gateway

### ISSUE THREAD

**Issue by leonardofrp5** [StateOfJS/StateOfJS-Vulcan#124]
![Captura de Pantalla 2022-01-17 a la(s) 14 20 42](https://user-images.githubusercontent.com/25190979/149826933-277763bd-daab-449f-91f8-d07b08b251e3.png)

**Comment by Mills-2** [comment]
This is the same issue as [this](https://github.com/StateOfJS/StateOfJS-homepage/issues/43). I would bet something is causing the survey to fail or its high traffic causing this.

**Comment by SachaG** [comment]
Yes, sorry about this. I restarted the server and it should be better now.

**Comment by superbiche** [comment]
504 Gateway Time-out now :grimacing: 
Thanks in advance :muscle:
